Domain Connect Spec Draft.adoc

```diff
@@ -709,6 +709,19 @@ EQ%3D%3D
 The Service Provider generates this signature using a private key. As indicated,
 this signature is generated from the query string properly URL encoded.
 
+The Service Provider must generate the signature with RSA256 hash,
+PKCS1v15 padding and Base64 encode it before adding it to query parameters.
+Example service in python can be found https://github.com/Domain-Connect/exampleservice/blob/master/sigutil.py[here]

```

Comment:
In the formal specification I would rather not refer to any example implementation or source code other than an example code snippet.